packages/pds/src/util/fs.ts

```diff
@@ -0,0 +1,7 @@
+import { RmOptions } from 'node:fs'
```

Comment:
Wouln't it be more consistent to use an import form `@atproto/common` here ? (meaning that we'd export `RmOptions` from `@atproto/common`)